src/payment/strategies/stripev3/stripev3-payment-strategy.spec.ts

```diff
@@ -575,48 +575,6 @@ describe('StripeV3PaymentStrategy', () => {
                     expect(stripeV3JsMock.confirmCardPayment).not.toHaveBeenCalled();
                 });
 
-                it('throws an error when paymentIntent.id is not set properly into payload and is Vaulted', async () => {
```

Comment:
I see that you remove some tests, and you add new code, why did not you add a tests that covers the new code?